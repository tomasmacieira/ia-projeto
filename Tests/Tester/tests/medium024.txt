ROW	3	1	5	0	1	3	1	1	4	1
COLUMN	1	3	3	3	1	1	0	0	7	1
2
HINT	0	2	L
HINT	8	8	M
