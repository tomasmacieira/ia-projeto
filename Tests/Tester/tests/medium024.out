..Lr....c.
.....c....
.lmr....lr
..........
........c.
lr..c.....
........t.
........m.
.lmr....M.
........b.
